### PR TITLE
Issue #66 point 3 fixed.

### PR DESCRIPTION
**Pull request by sagarmulchandani**
Related issues: https://github.com/Umuzi-org/ACN-syllabus/issues/66

## Description:

What are you up to? Fill us in :)
Fixed task no 3.) mentioned in the [issue](https://github.com/Umuzi-org/ACN-syllabus/issues/66).
Updated  the clone repo link in `projects/git-exercises/_index.md`
Fixed a spelling mistake in `content/topics/git/_index.md`

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by sagarmulchandani** [comment]
This is part of my contribution towards `Hacktoberfest`. So, if you find my PR legible please to add a `hacktoberfest-approved label` to it. Thank you :)

File: content/projects/git-exercises/_index.md [hunk 1]
```diff
@@ -341,7 +341,7 @@ Go look at your network page. each of the dots there represent commits you made.
 2. Now let's clone a repo. point your browser here: https://github.com/Umuzi-org/a-repo-to-clone
 3. Now there is a friendly green button that says "Clone or download". Click on it.
 4. You will see a url come up. Copy it. You will need to paste it into the terminal in a moment
-5. In your terminal type in `git clone $THE_URL_YOU_JUST_COPIED`. It should look something like this: `git clone https://github.com/Umuzi-org/tech-department`
+5. In your terminal type in `git clone $THE_URL_YOU_JUST_COPIED`. It should look something like this: `git clone https://github.com/Umuzi-org/a-repo-to-clone`
 6. `cd` into the tech-department directory that was just created
```
Comment (ry-oc) [inline content/projects/git-exercises/_index.md:345]: Please also correct this line to refer to the corrected directory

**Comment by ry-oc** [comment]
Hi @sagarmulchandani are you only going to be addressing point 3 of this issue or do you also intend to address the other points?

File: content/projects/git-exercises/_index.md (hunk 1 above)
Comment (sagarmulchandani) [reply content/projects/git-exercises/_index.md]: This is fixed

**Comment by sagarmulchandani** [comment]
Hi @ry-oc , 
I have made the changes you suggested. 
Regarding the other 2 points mentioned in the [issue](https://github.com/Umuzi-org/ACN-syllabus/issues/66), they are already fixed. Let me know if anything else needs to be changed. 

Also, please do add a `hacktoberfest-approved` label to the PR if you think its legible. Thank you :)

File: content/projects/git-exercises/_index.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ This little exercise will take you through the basic git mechanisms you need to
 
 **IMPORTANT** Make sure you go through the prerequisites listed above. Otherwise you'll be clueless!
 
-## Skillz you'll learn
+## Skills you'll learn
```
Comment (ry-oc) [inline content/projects/git-exercises/_index.md:26]: @sagarmulchandani please revert this change as the spelling was intentional, as soon as this is done i can approve :)

Comment (sagarmulchandani) [reply content/projects/git-exercises/_index.md]: Done. You can go ahead with the review

**Review by ry-oc — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**